web/src/features/fireWeather/components/graphs/TempRHGraph.tsx

```diff
@@ -161,13 +161,35 @@ const TempRHGraph = (props: Props) => {
 
   // Update plotly revision to trigger re-drawing of the plot
   const setRevision = useState(0)[1]
+  const [hoverMode, setHoverMode] = useState<
+    'closest' | 'x' | 'y' | 'x unified' | 'y unified' | false
+  >('closest')
+
+  const handleHoverModeChange = (
+    event: React.ChangeEvent<{ name?: string; value: string }>
+  ) => {
+    switch (event.target.value) {
+      case 'closest':
+      case 'x':
+      case 'x unified':
+        setHoverMode(event.target.value)
+    }
```

Comment:
do we want to use Matomo to see which hover mode is used most often?

Reply:
In theory, but really our sample size is so small.....

Reply:
I'm going to skip adding instrumentation.